remix.init/remix.config.js

```diff
@@ -2,7 +2,7 @@ const { config } = require("@netlify/remix-edge-adapter");
 const baseConfig =
   process.env.NODE_ENV === "production"
     ? config
-    : { ignoredRouteFiles: ["**/.*"] };
+    : { ignoredRouteFiles: ["**/.*"], future: config.future };
```

Comment:
Pulling this is so that the new routing works for local development as well for the Remix Netlify Edge Functions template.